operations/operations/publishing/publishing-guidelines/writing-an-mvp.md

```diff
@@ -0,0 +1,41 @@
+# Writing an MVP
+
+Our technical documentation lives at [docs.mattermost.com](https://docs.mattermost.com), primarily written using ReStructured Text, using Sphinx and Write The Docs as our build and theme tools. 
```

Comment:
I would say maybe say "admin documentation" or potentially both admin and technical.

Reply:
Good point. What about "admin, user, and technical documentation"?

Reply:
That sounds good.